packages/ui-source-code-editor/src/SourceCodeEditor/index.tsx

```diff
@@ -483,7 +483,7 @@ class SourceCodeEditor extends Component<SourceCodeEditorProps> {
     const { styles } = this.props
 
     if (!styles?.theme || !styles.highlightStyle) {
-      return
+      return undefined
```

Comment:
Why is this needed here?

Reply:
ESLint expects that a getter always returns a value. IMO this is a good rule, and its OK if we make such edge cases more explicit